tinder.js

```diff
@@ -63,9 +66,32 @@ function TinderClient() {
    * @param {Object} data an object containing extra values
    * @param {Function} callback the callback to invoke when the request completes
    */
-  var tinderPost = function(path, data, callback) {
-    var opts = getRequestOptions(path, data);
+  var tinderPost = function(path, data, callback, file) {
```

Comment:
This shouldn't be needed, the data object should be used here (data['file']). Rather than use fs to take a path from the filesystem, whatever's calling the library should supply the file...